RecoPPS/Local/src/TotemTimingRecHitProducerAlgorithm.cc

```diff
@@ -88,7 +88,7 @@ void TotemTimingRecHitProducerAlgorithm::build(const CTPPSGeometry& geom,
       auto max_corrected_it = std::max_element(dataCorrected.begin(), dataCorrected.end());
 
       float t = TotemTimingRecHit::NO_T_AVAILABLE;
-      if (det->name() == "CTPPS_Diamond_Segment" && (*min_it) > saturationLimit_ || (*max_it) < saturationLimit_)
+      if (det->name() == ("CTPPS_Diamond_Segment" && (*min_it) > saturationLimit_) || (*max_it) < saturationLimit_)
```

Comment:
```suggestion
      if (det->name() == "CTPPS_Diamond_Segment" && ((*min_it) > saturationLimit_ || (*max_it) < saturationLimit_))
```
shouldn't it be this instead?

Reply:
Sorry for the confusion, should be fine now.